profiler/test/Datadog.Profiler.IntegrationTests/SingleStepInstrumentation/SingleStepInstrumentationTest.cs

```diff
@@ -0,0 +1,463 @@
+        {
+            var runner = new TestApplicationRunner(appName, framework, appAssembly, _output, commandLine: "--scenario 1", enableProfiler: false, enableTracer: true);
+
+            // deployed with SSI
+            runner.Environment.SetVariable(EnvironmentVariables.SsiDeployed, "profiler");
+            // simulate long lived
+            runner.Environment.SetVariable(EnvironmentVariables.SsiShortLivedThreshold, "0");
+
+            using var agent = MockDatadogAgent.CreateHttpAgent(_output);
+
+            List<Serie> nbProfilesSeries = [];
+            Serie runtimeIdSerie = null;
+            agent.TelemetryMetricsRequestReceived += (_, ctx) =>
+            {
+                var s = GetRequestText(ctx.Value.Request);
+
+                var (nbRuntimeId, nbProfiles) = ExtractSeries(s);
+
+                if (nbRuntimeId != null && runtimeIdSerie != null)
+                {
+                    Assert.Fail("There must be only one 'ssi_heuristic.number_of_runtime_id' serie'");
+                }
+
+                runtimeIdSerie = nbRuntimeId;
+                nbProfilesSeries.AddRange(nbProfiles);
+            };
+
+            runner.Run(agent);
+
+            runtimeIdSerie.Should().NotBeNull();
+            var expectedTags = new[] { "has_sent_profiles:true", "heuristic_hypothetical_decision:triggered", "installation:ssi", "enablement_choice:ssi_enabled" };
+            runtimeIdSerie.Tags.Should().BeEquivalentTo(expectedTags);
+
+            var hasTriggeredSeries = false;
+            nbProfilesSeries.Should().AllSatisfy(s =>
+            {
+                s.Metric.Should().Be("ssi_heuristic.number_of_profiles");
+                s.Tags.Should().HaveCount(4).And.OnlyHaveUniqueItems().And.Contain("installation:ssi", "enablement_choice:ssi_enabled");
+                if (s.Tags.Contains("has_sent_profiles:true"))
+                {
+                    s.Tags.Should().Contain("heuristic_hypothetical_decision:triggered");
+                    hasTriggeredSeries = true;
+                }
+            });
+
+            hasTriggeredSeries.Should().BeTrue("We must have some triggered serie(s)");
+
+            agent.NbCallsOnProfilingEndpoint.Should().NotBe(0);
+        }
+
+        [TestAppFact("Samples.BuggyBits")]
+        public void CheckSsiDeployedAndProfilingSsiEnabled_Delayed(string appName, string framework, string appAssembly)
+        {
+            var runner = new TestApplicationRunner(appName, framework, appAssembly, _output, commandLine: "--scenario 1", enableProfiler: false, enableTracer: true);
+
+            // deployed with SSI
+            runner.Environment.SetVariable(EnvironmentVariables.SsiDeployed, "profiler");
+            // simulate long lived
+            runner.Environment.SetVariable(EnvironmentVariables.SsiShortLivedThreshold, TimeSpan.FromSeconds(6).TotalMilliseconds.ToString());
```

Comment:
The 3 long lived tests are using different values (1ms, 0ms and 6000ms here)
Maybe the same value should be consistently used